cdap-app-fabric/src/main/java/io/cdap/cdap/internal/operations/OperationRunFilter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.operations;
+
+import io.cdap.cdap.proto.operationrun.OperationRunStatus;
+import javax.annotation.Nullable;
+
+/**
+ * This class defined various filters that can be applied during operation runs scanning.
+ */
+public class OperationRunFilter {
```

Comment:
Usually a Filter is kind of a Function instead of just having getters. How is it supposed to be used?

Reply:
It is just a container class to encapsulate what filter options are available for operations.